util-core/src/main/scala/com/twitter/util/Try.scala

```diff
@@ -1,5 +1,7 @@
 package com.twitter.util
 
+import scala.util.{ Success, Failure }
```

Comment:
style nit: drop the extra spaces around the imports. Alternatively, we can fully qualify these at the usage site to be explicit.